pkg/expr/sql_command.go

```diff
@@ -121,3 +135,13 @@ func (gr *SQLCommand) Execute(ctx context.Context, now time.Time, vars mathexp.V
 func (gr *SQLCommand) Type() string {
 	return TypeSQL.String()
 }
+
+func totalRows(frames []*data.Frame) int64 {
```

Comment:
Nit, for something this short I would go for a named return val.
```suggestion
func totalRows(frames []*data.Frame) (total int64) {
```

Reply:
I can do this. Can you explain why you prefer it?

Reply:
I don't really care, just my impression of what is more "idiomatic", could be wrong though.

Reply:
ok thanks. I made the change anyway :-)